src/claims/bold/intellij/avro/idl/AvroIdl.bnf

```diff
@@ -95,7 +95,7 @@ enum_value ::= IDENTIFIER
 fixed_decl ::= annotation* 'fixed' IDENTIFIER '(' INT_LITERAL ')' ';' { pin = 2 }
 
 message_decl ::= type IDENTIFIER '(' arguments ')' message_attributes* ';' { pin = 3 }
-private message_attributes ::= 'throws' IDENTIFIER | 'oneway' { pin = 1}
+private message_attributes ::= ( 'throws' IDENTIFIER (',' IDENTIFIER)* ) | 'oneway' { pin = 1}
```

Comment:
Does explicitly grouping the left-hand of the pipe in `()` affect the `pin` attribute? I'm rusty with the inner workings of the grammar.

Reply:
I'm not sure. The parser behaves as if `throws` or `oneway` are pinned, so I guess it works. 
I tried changing it and didn't see any changes in the generated parser code.

I can split this rule in two if you think it helps:
```
message_decl ::= type IDENTIFIER '(' arguments ')' (throws_message_attribute | oneway_message_attribute)* ';' { pin = 3 }
private throws_message_attribute ::= 'throws' IDENTIFIER (',' IDENTIFIER)* { pin = 1 }
private oneway_message_attribute ::= 'oneway'
```